packages/@aws-cdk/aws-applicationautoscaling/package.json

```diff
@@ -24,6 +24,9 @@
         "distName": "aws-cdk.aws-applicationautoscaling",
         "module": "aws_cdk.aws_applicationautoscaling"
       }
+    },
+    "metadata": {
+      "awsdocs:title": "Application Auto Scaling"
```

Comment:
Should this be [AWS AutoScaling](https://docs.aws.amazon.com/autoscaling/index.html)?